website/content/grants/contents+en.lr

```diff
@@ -0,0 +1,84 @@
+title: Grant Program
+---
+body:
+
+The Python Software Verband e.V. (German Python Software Association) offers a funding program to financially support
+the Python community in German-speaking countries in their diverse activities.
+
```

Comment:
Same here for German-speaking ... (see German version)

Reply:
removed "German"